### ISSUE TITLE
State parameter length issue when working with spring security oauth2

### ISSUE THREAD

**Issue by pragkent** [ory/fosite#267]
The default state generator of spring security oauth2 library generates state with length of 6 which is less than the `MinParameterEntropy`(which is 8).  [check code here](https://github.com/spring-projects/spring-security-oauth/blob/master/spring-security-oauth2/src/main/java/org/springframework/security/oauth2/common/util/RandomValueStringGenerator.java) 

I wonder that if it's possible to adjust MinParameterEntropy? or make it configurable?

**Comment by arekkas** [comment]
Making it configurable would be an option

**Comment by pragkent** [comment]
Any thought of how to implement it?
My first thought was making MinParameterEntropy a variable instead of a const.

**Comment by arekkas** [comment]
Yeah, just add a `Fosite.GetMinParameterEntropy()` function which returns either `MinParameterEntropy`, or - if set - the value of e.g. `Fosite.MinParameterEntropy` which can be set when initializing the struct.

**Comment by mitar** [comment]
I made https://github.com/ory/fosite/pull/461 to address this.
